traffic_ops/traffic_ops_golang/trafficvault/backends/postgres/encrypt.go

```diff
@@ -0,0 +1,92 @@
+package postgres
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"errors"
+	"io"
+	"io/ioutil"
+)
+
+func encryptString(stringToEncrypt string, encryptionKeyLocation string) (string, error) {
+	key, err := readKeyFromFile(encryptionKeyLocation)
```

Comment:
Thinking about fetching this key file from HashiCorp Vault, we probably wouldn't want to fetch it every time we needed to encrypt/decrypt. I think the same would apply to on-disk storage. We probably just want to read it into memory during startup (in `postgresLoad()`) and save it as a private field in the `Postgres` struct in order to reuse it.